Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2_test.py

```diff
@@ -19,6 +19,15 @@
     ensure_aware,
 )
 from CommonServerPython import GetModifiedRemoteDataResponse
+from CybleEventsV2 import check_response
+
+
+def test_test_response_success(mock_client):
+    mock_client._http_request = Mock(return_value={"status": "ok"})
+    with patch("CybleEventsV2.demisto", demisto_mock):
+        result = check_response(mock_client, "GET", "https://example.com", "dummy-token")
+    assert result == "ok"
+
```

Comment:
To ensure complete test coverage, please add a unit test for the failure scenario of the `check_response` function. This test should verify that an exception is raised when the function receives an empty or `None` response from the `_http_request` call.